CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - New parameters in the PGI (`pgi`) building block
 - Internal reorganization of files into sub-directories for building
   blocks, templates, and primitives
+- Added `user`-primitive for setting the active user in Dockerfiles, usage: `Stage0 += user(user='root')`
```

Comment:
This should go in a section for version 18.8.0.  I can take care of this when I prepare the next release, so please just revert this change.